### PR TITLE
Fixed CODEOWNERS for USM

### PR DESCRIPTION
**Pull request by guyarb**
### What does this PR do?

- Add go-tls files to CODEOWNERS file
- Split wrong location of USM definitions inside Networks files

### Reviewer's Checklist

- [ ] If known, an appropriate milestone has been selected; otherwise the `Triage` milestone is set.
- [ ] Use the `major_change` label if your change either has a major impact on the code base, is impacting multiple teams or is changing important well-established internals of the Agent. This label will be use during QA to make sure each team pay extra attention to the changed behavior. For any customer facing change use a releasenote.
- [ ] A [release note](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md#reno) has been added or the `changelog/no-changelog` label has been applied.
- [ ] Changed code has automated tests for its functionality.
- [ ] Adequate QA/testing plan information is provided if the `qa/skip-qa` label is not applied.
- [ ] At least one `team/..` label has been applied, indicating the team(s) that should QA this change.
- [ ] If applicable, docs team has been notified or [an issue has been opened on the documentation repo](https://github.com/DataDog/documentation/issues/new).
- [ ] If applicable, the `need-change/operator` and `need-change/helm` labels have been applied.
- [ ] If applicable, the `k8s/<min-version>` label, indicating the lowest Kubernetes version compatible with this feature. 
- [ ] If applicable, the [config template](https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml) has been updated.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -276,6 +276,9 @@
 /pkg/network/ebpf/c/prebuilt/http*      @DataDog/universal-service-monitoring
 /pkg/network/ebpf/c/runtime/http*       @DataDog/universal-service-monitoring
 /pkg/network/ebpf/c/http*               @DataDog/universal-service-monitoring
+/pkg/network/ebpf/c/go-tls-*            @DataDog/universal-service-monitoring
+/pkg/network/ebpf/c/tags-types.h        @DataDog/universal-service-monitoring
+/pkg/network/ebpf/go_tls*              @DataDog/universal-service-monitoring
```
Comment (Yumasi) [inline .github/CODEOWNERS:281]: Wrong formatting here.

File: pkg/network/tags_linux.go [hunk 2]
```diff
@@ -9,12 +9,12 @@
...
 
 import (
-	netebpf "github.com/DataDog/datadog-agent/pkg/network/ebpf"
+	"github.com/DataDog/datadog-agent/pkg/network/http"
 )
 
 // GetStaticTags return the string list of static tags from network.ConnectionStats.Tags
 func GetStaticTags(staticTags uint64) (tags []string) {
-	for tag, str := range netebpf.StaticTags {
+	for tag, str := range http.StaticTags {
```
Comment (brycekahle) [inline pkg/network/tags_linux.go:17]: What happens when you have tags that are not HTTP-specific, or even USM-specific?

**Review by Yumasi — COMMENTED** [review]
Formatting in CODEOWNERS.

File: pkg/network/tags_linux.go (hunk 2 above)
Comment (guyarb) [reply pkg/network/tags_linux.go:17]: ATM, there are no tags besides those
so if we will have more we should add it here
WDYT?

File: .github/CODEOWNERS (hunk 1 above)
Comment (Yumasi) [reply .github/CODEOWNERS]: Fixed in [09fd23a](https://github.com/DataDog/datadog-agent/pull/13852/commits/09fd23acc9ef1afaedbf3a8edbdbe4aa53f9f075)

**Review by hmahmood — APPROVED** [review]

**Review by Yumasi — APPROVED** [review]

File: pkg/network/tags_linux.go (hunk 2 above)
Comment (hmahmood) [reply pkg/network/tags_linux.go:17]: Maybe every package can have its own tags? We would have to go over each collection separately here. Downside is that we don't have them in one place so there could be (unintended) duplication.

Comment (guyarb) [reply pkg/network/tags_linux.go:17]: They will add it to their package and they will add here the iteration on the tags
Or what hasan proposed seems good as well

Comment (guyarb) [reply pkg/network/tags_linux.go:17]: Do you have a suggestion to make it better?

Comment (brycekahle) [reply pkg/network/tags_linux.go:17]: I think the original intent of having it `pkg/network/ebpf` was to have a common place, but your idea(s) seem fine. We can always iterate on it, in the future.

**Review by brycekahle — APPROVED** [review]

**Pull request merged**